### DISCUSSION TITLE
Taking ages to install on Gitpod

### DISCUSSION THREAD

**Discussion by aksthelion** [forem/forem#20420] [Q&A]
Hello, 
I'm trying to install it on Gitpod and it is taking ages. It has been almost 4 hours since I started installing. Not sure what is happening. Does it so long just to install? There are no problems reported. It is just so slow. Any help? Thanks.

**Comment by Justman100** [comment]
> Hello, I'm trying to install it on Gitpod and it is taking ages. It has been almost 4 hours since I started installing. Not sure what is happening. Does it so long just to install? There are no problems reported. It is just so slow. Any help? Thanks.

Do you have a screen of the process?

**Comment by Shubham00043** [comment]
Long install times on Gitpod can sometimes happen because the workspace is still downloading large dependencies or building containers in the background.

You can try restarting the workspace and rebuilding dependencies from scratch. If possible, also check the terminal logs to see which step is taking the longest.

Sometimes removing cached dependencies and reinstalling helps too:

```bash id="p4n8x2"
rm -rf node_modules
npm install
```

If Docker/dev containers are being used, rebuilding the container may also fix stuck installs.
